### PR TITLE
feature (elmo_model_extended_configs): add new configs, fix bag of the st…

### PR DESCRIPTION
**Pull request by kudep**
Update configs of elmo LM model.

### CODE REVIEW

File: deeppavlov/models/elmo/elmo.py [hunk 1]
```diff
@@ -103,10 +103,34 @@ class ELMo(NNModel):
...
+    ``elmo-lm-ready4fine-tuning-ru-wiki`` configuration file or ``elmo-lm-ready4fine-tuning-ru-wiki-simple``
+    configuration file.
+
+    `simple` configuration file is a configuration of a model without special tags of output
+    vocab used for first training.
+
+    .. note::
+
+        You need to download about **4 GB** also by default about **32 GB** of RAM and **10 GB** of GPU memory
+        required to running the configuration ``deeppavlov/configs/elmo/elmo-lm-ready4fine-tuning-ru-*.json``
```
Comment (yoptar) [inline deeppavlov/models/elmo/elmo.py]: Are the requirements the same for `simple` configs?

Comment (kudep) [reply deeppavlov/models/elmo/elmo.py]: Yes

File: docs/intro/pretrained_vectors.rst [hunk 2]
```diff
@@ -5,11 +5,31 @@ ELMo
 ----
 
 | We are publishing :class:`Russian language ELMo embeddings model <deeppavlov.models.embedders.elmo_embedder.ELMoEmbedder>` for tensorflow-hub.
+| We are publishing :class:`LM model <deeppavlov.models.elmo.elmo.ELMo>` for training and fine-tuning ELMo as LM model.
```
Comment (nikolay-bushkov) [inline docs/intro/pretrained_vectors.rst]: Maybe avoid word duplication?

File: deeppavlov/models/elmo/elmo.py [hunk 3]
```diff
@@ -127,7 +151,7 @@ class ELMo(NNModel):
         .. note::
 
             You need to download about **2 GB** also by default about **10 GB** of RAM and **10 GB** of GPU memory
-            required to running the config ``deeppavlov/configs/elmo/elmo-1b-benchmark.json`` on one GPU.
+            required to running the configuration ``deeppavlov/configs/elmo/elmo-1b-benchmark.json`` on one GPU.
```
Comment (nikolay-bushkov) [inline deeppavlov/models/elmo/elmo.py]: There is special :config: markup for DeepPavlov configs.

**Pull request merged**